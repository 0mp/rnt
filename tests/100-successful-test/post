. ../post.functions

assert_nonexistent "test/err.actual"
assert_nonexistent "test/err.diff"
assert_nonexistent "test/err.diff.tmp"

assert_nonexistent "test/exit.actual"
assert_nonexistent "test/exit.diff"
assert_nonexistent "test/exit.diff.tmp"

assert_nonexistent "test/other.actual"
assert_nonexistent "test/other.diff"
assert_nonexistent "test/other.diff.tmp"

assert_nonexistent "test/out.actual"
assert_nonexistent "test/out.diff"
assert_nonexistent "test/out.diff.tmp"
