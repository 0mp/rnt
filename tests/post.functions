cleanup()
{
  rm -f "$PWD/test/exit.actual"
  rm -f "$PWD/test/err.actual"
  rm -f "$PWD/test/out.actual"
  rm -f "$PWD/test/exit.diff"
  rm -f "$PWD/test/err.diff"
  rm -f "$PWD/test/out.diff"
}

assert_file()
{
  if test -e "${1:?}"; then
    echo "present: $1"
  else
    echo "missing: $1"
  fi
}

assert_nonexistent()
{
  assert_file ${1+"$@"}
}

assert_same()
{
  diff -u "${1:?}" "${2:?}"
}

