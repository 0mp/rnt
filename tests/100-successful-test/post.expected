missing: test/err.actual
missing: test/err.diff
missing: test/err.diff.tmp
missing: test/exit.actual
missing: test/exit.diff
missing: test/exit.diff.tmp
missing: test/other.actual
missing: test/other.diff
missing: test/other.diff.tmp
missing: test/out.actual
missing: test/out.diff
missing: test/out.diff.tmp
